config/core/deployments/broker.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: broker-ingress
+  namespace: cloud-run-events
+  labels:
+    events.cloud.google.com/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: cloud-run-events
+      role: broker-ingress
```

Comment:
Maybe do similar things as https://github.com/knative/eventing/blob/master/pkg/reconciler/broker/resources/ingress.go#L155-L160

Reply:
I am using the same style as other deployments in `knative-gcp`. I think we should be consistent with `knative-gcp`. What do you tink?

Reply:
I didn't realize that. I agree keeping it consistent at the moment is a better idea.